src/model/entities/piece.ts

```diff
@@ -135,6 +138,14 @@ export class Piece {
     return this.duration
   }
 
+  public isInsertedOnAir(): boolean {
+    return this.isPieceInsertedOnAir
+  }
+
+  public markAsInsertedOnAir(): void {
+    this.isPieceInsertedOnAir = true
+  }
```

Comment:
Only unplanned Pieces can be inserted, so it would be nice to have guard against marking planned Pieces as inserted on Air